frontend/modules/store.ts

```diff
@@ -14,7 +14,7 @@ export default function configureStore(initialState = {}) {
 	const enhancers = [];
 
 	if (IS_DEVELOPMENT) {
-		middlewares.unshift(require('redux-immutable-state-invariant').default());
+		// middlewares.unshift(require('redux-immutable-state-invariant').default());
```

Comment:
What is this line for?

Reply:
I will uncomment this line after migrating RevisionsService, because now errors are unnecessarily displayed by this library